Add bdb ic list command & refactor documentation of bdb ic cmd [KRKNWK-11621] (#5892)
nrfconnect/sdk-nrf: doc/nrf/libraries/zigbee/shell.rst

----

.. _bdb_ic_add:

bdb ic add
==========

Add information about the install code on the trust center.

.. parsed-literal::
   :class: highlight

   bdb ic add *h:install code* *h:eui64*

For *h:eui64*, use the address of the joining device.

.. note::
    |precondition3|

    |precondition5|

    |precondition6|

    |precondition7|

Example:

.. code-block::

   > bdb ic add 83FED3407A939723A5C639B26916D505C3B5 0B010E2F79E9DBFA
   Done

----

.. _bdb_ic_list:

bdb ic list
===========

> Reads and prints install codes stored on the device.

Reviewer: ```suggestion
Read and print install codes stored on the device.
```

.. parsed-literal::
   :class: highlight

   bdb ic list

.. note::
    |precondition4|

    |precondition6|

    |precondition7|

Example:

.. code-block::

   > bdb ic list
   [idx] EUI64:           IC:                                  options:
   [  0] 0b010e2f79e9dbfa 83fed3407a939723a5c639b26916d505c3b5 0x3
   Total entries for the install codes table: 1
   Done

----

.. _bdb_ic_policy:

bdb ic policy
=============

Set the trust center install code policy.

.. parsed-literal::
   :class: highlight

>    bdb ic policy *enable|disable*

Reviewer: No example for this one?

.. |precondition3| replace:: Adding install codes only after :ref:`bdb_start`.

.. |precondition4| replace:: Use only after :ref:`bdb_start`.

> .. |precondition5| replace:: Provide the install code as an ASCII-encoded :file:`HEX` file that includes CRC16/X-25 in little-endian order.

Reviewer: ```suggestion
.. |precondition5| replace:: Provide the install code as an ASCII-encoded hexadecimal file that includes CRC16/X-25 in little-endian order.
```

> .. |precondition6| replace:: For production devices, an install code must be installed by the production configuration present in flash.

Reviewer: ```suggestion
.. |precondition6| replace:: For production devices, make sure that an install code is installed by the production configuration present in the flash.
```
